include/inviwo/core/datastructures/transferfunction.h

```diff
@@ -128,4 +129,31 @@ class IVW_CORE_API TransferFunction : public TFPrimitiveSet {
 bool operator==(const TransferFunction& lhs, const TransferFunction& rhs);
 bool operator!=(const TransferFunction& lhs, const TransferFunction& rhs);
 
+template <>
+struct DataTraits<TransferFunction> {
+    static std::string classIdentifier() { return "org.inviwo.transferfunction"; }
```

Comment:
one can return const std::string& here to make it a bit more performant

Reply:
Ah, I updated this and the comment in DataTraits to reflect it. Also updated DataTraits<DataFrame> since I "stole" most of the code from there :)